sklearn/metrics/tests/test_pairwise.py

```diff
@@ -4,8 +4,16 @@
 from numpy import linalg
 
 from scipy.sparse import dok_matrix, csr_matrix, issparse
-from scipy.spatial.distance import cosine, cityblock, minkowski, wminkowski
+from scipy.spatial.distance import cosine, cityblock, minkowski
 from scipy.spatial.distance import cdist, pdist, squareform
+try:
+    from scipy.spatial.distance import wminkowski
+except ImportError:
+    # In scipy 1.6.0, wminkowski is deprecated and minkowski
+    # should be used instead.
+    from scipy.spatial.distance import minkowski as wminkowski
```

Comment:
As for coverage, the `ImportError` will not trigger until scipy 1.8 is released and `wminkowski` is removed.

(This is a comment: no changes required here)